src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -91,6 +91,7 @@ class HoldRequest extends React.Component {
     let path = `${appConfig.baseUrl}/hold/confirmation/${bibId}-${itemId}`;
     const searchKeywordsQuery =
       (this.props.searchKeywords) ? `searchKeywords=${this.props.searchKeywords}` : '';
+    const searchKeywordsQueryPhysical = searchKeywordsQuery ? `&${searchKeywordsQuery}` : '';
 
     if (this.state.delivery === 'edd') {
       const searchKeywordsQueryEdd = searchKeywordsQuery ? `?${searchKeywordsQuery}` : '';
```

Comment:
I don't mind if it means one less variable and you can miss why those variable are different. But you can also miss how the `&` and `?` in the URL endpoint. So either way, we just have to pay closer attention. Up to you.